xbmc/guilib/guiinfo/GUIInfoLabel.h

```diff
@@ -1,10 +1,10 @@
 /*!
-\file GUIInfoTypes.h
+\file GUIInfoLabel.h
 \brief
 */
 
-#ifndef GUILIB_GUIINFOTYPES_H
-#define GUILIB_GUIINFOTYPES_H
+#ifndef GUILIB_GUIINFOLABEL_H
```

Comment:
There is `#pragma once` below. Just remove this.

Reply:
OK, I tried to preserve the current coding style - but I'll remember to apply such changes.

Reply:
The problem is that a lot of the existing code is quite old and does not conform to our own code guidelines. But we do want new code and clean-up effort to follow them. While there is no hard rule on this, I think `#pragma once` is the established convention now. In any case, having both include guards and `#pragma once` makes zero sense even if it was that way before :-) Unless someone disagrees please use the pragma only.